odcockpit/src/plugins/environmentviewer/EnvironmentViewerGLWidget.cpp

```diff
@@ -522,4 +520,4 @@ namespace cockpit {
             }
         }
     }
-} // plugins::environmentviewer
+} // plugins::environmentviewer銏聺籲籼x腻究|蚀硳}穷z要硵y誀籼z趇癴w誃
```

Comment:
@HangYin2015 Could you please revert the changes to this file so that I can complete the PR?